packages/primitives/progress-bar/src/Root.native.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { View } from 'react-native'
+import type { ViewProps } from 'react-native'
+import { normalizeNativeStyle } from 'stili'
+import type { TStyle } from 'stili'
+import { startWithType, component, mapProps } from 'refun'
+import { isNumber } from 'tsfn'
+import type { TProgressBar } from './types'
+
+export const ProgressBar = component(
+  startWithType<TProgressBar>(),
+  mapProps(
+    ({
+      style, id, ariaValuenow, ariaValuemin, ariaValuemax, children,
+    }) => {
+      const styles: TStyle = {
+        borderStyle: 'solid',
+        borderWidth: 0,
+        position: 'relative',
+        flexDirection: 'row',
+        flexGrow: 0,
+        flexShrink: 0,
+        alignSelf: 'flex-start',
+        ...style,
+      }
+      const props: ViewProps = {
+      }
+
+      if (typeof id === 'string') {
+        props.testID = id
+      }
+
+      if (isNumber(ariaValuenow)) {
+        props.accessibilityValue = {
+          ...(isNumber(ariaValuemin) ? { min: ariaValuemin } : {}),
+          now: ariaValuenow,
+          ...(isNumber(ariaValuemax) ? { max: ariaValuemax } : {}),
+        }
+        props.accessible = true
```

Comment:
Is it so simple? 😂 Just passing props `accessible = true` and we're done here? :trollface:

Reply:
yes!
actually my next PR is just adding it to all components and thats it!

this prop makes the screen reader to know it should take a look here, without it the screen reader do not read the `accessibilityValue`